src/app/App.tsx

```diff
@@ -12,7 +12,7 @@ import RoutineDetails from './routines/routineDetails/RoutineDetails';
 import RoutineCreate from './routines/routineForm/routineCreate/RoutineCreate';
 import RoutineUpdate from './routines/routineForm/routineUpdate/RoutineUpdate';
 
-import '@/App.scss';
+import WorkoutCreate from './workouts/workoutCreate/WorkoutCreate.tsx';
```

Comment:
better name it "NewWorkout"

Reply:
Completely agree! (thats what I wanted to do)
But the problem is that we will have a naming conflict:
<img width="501" alt="image" src="https://github.com/user-attachments/assets/46a95fc3-d2a2-4149-9f82-09d590871e02">
what do you think about it?

Reply:
ye, can be fine with that naming convention. 
Just to have proper routing for it so it fits Our layout -> '/my-trainings/new'